tests/test_interface.py

```diff
@@ -88,6 +86,51 @@ def test_InterfaceChangeMtu(self, dvs):
                     if fv[0] == "SAI_ROUTER_INTERFACE_ATTR_MTU":
                         assert fv[1] == "8888"
 
+    def test_InterfaceAddRemoveSecondIpv4Address(self, dvs):
+        pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+        adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+
+        # assign second IP to interface
+        tbl = swsscommon.Table(cdb, "INTERFACE")
+        fvs = swsscommon.FieldValuePairs([("NULL", "NULL")])
+        tbl.set("Ethernet8|20.0.0.4/31", fvs)
```

Comment:
by default each test should be independent, so that when a test fails, we can debug it and re-run it.